lib/liquid/version.rb

```diff
@@ -1,4 +1,4 @@
 # encoding: utf-8
 module Liquid
-  VERSION = "4.0.0.rc2"
+  VERSION = "4.0.0.rc2".freeze
```

Comment:
This was removed on purpose in https://github.com/Shopify/liquid/pull/371, although maybe that is ok now that we don't seem to support ruby 1.9 now.